### PR TITLE
PSFR1922 - Assign a case whats new and refactor

### PR DESCRIPTION
**Pull request by sturner-HippoDigital**
Refactored the code for easier updates to this going forward.

### CODE REVIEW

File: server/banner.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+import logger from '../logger'
+import { BannerFile } from './@types/express'
+
+export default class Banner {
```
Comment (JamesBoobier) [inline server/banner.ts:6]: Could you add some unit tests for this class?

File: server/views/pages/service-updates.njk [hunk 2]
```diff
@@ -2,26 +2,25 @@
 {% set pageName = "Service updates" %}
 
 {% block content %}
-  {% if whatsNewVersion >= 20241120 %}
```
Comment (JamesBoobier) [inline server/views/pages/service-updates.njk:5]: Are we still showing only the banners after the version in the config in here?

Comment (sturner-HippoDigital) [reply server/views/pages/service-updates.njk:5]: Yes now it filters the list in the serviceUpdateController.ts to only versions less than or equal to the version in the config

File: server/banner.ts (hunk 1 above)
Comment (sturner-HippoDigital) [reply server/banner.ts:6]: Added tests

**Review by JamesBoobier — APPROVED** [review]

**Pull request merged**